cider/tests/utils/CiderTestBase.cpp

```diff
@@ -50,12 +50,26 @@ void CiderTestBase::prepareArrowBatch() {
 void CiderTestBase::assertQueryArrow(const std::string& sql,
                                      const std::string& json_file) {
   auto duck_res = duckDbQueryRunner_.runSql(sql);
-  auto duck_res_batch = DuckDbResultConvertor::fetchDataToCiderBatch(duck_res);
+  auto duck_res_batch =
+      DuckDbResultConvertor::fetchDataToArrowFormattedCiderBatch(duck_res);
+
+  auto cider_input = json_file.size() ? json_file : sql;
+  auto cider_res_batch = std::make_shared<CiderBatch>(
+      ciderQueryRunner_.runQueryOneBatch(cider_input, input_[0], true));
+  EXPECT_TRUE(CiderBatchChecker::checkArrowEq(duck_res_batch, cider_res_batch, false));
+}
+
+void CiderTestBase::assertQueryArrowIgnoreOrder(const std::string& sql,
+                                                const std::string& json_file) {
+  auto duck_res = duckDbQueryRunner_.runSql(sql);
+  auto duck_res_batch =
+      DuckDbResultConvertor::fetchDataToArrowFormattedCiderBatch(duck_res);
 
   auto cider_input = json_file.size() ? json_file : sql;
   auto cider_res_batch = std::make_shared<CiderBatch>(
       ciderQueryRunner_.runQueryOneBatch(cider_input, input_[0], true));
-  EXPECT_TRUE(CiderBatchChecker::checkArrowEqTemp(duck_res_batch, cider_res_batch));
+
+  EXPECT_TRUE(CiderBatchChecker::checkEq(duck_res_batch, cider_res_batch, true));
```

Comment:
I think it probably should be `EXPECT_TRUE(CiderBatchChecker::checkArrowEq(...))` here?

Reply:
Yes, just the same as above, let me revise it.